src/main/resources/native/android/android_project/app/build.gradle

```diff
@@ -2,15 +2,14 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 27
-    buildToolsVersion "27.0.3"
 
     defaultConfig {
         minSdkVersion 21
         targetSdkVersion 27
```

Comment:
According to https://developer.android.com/studio/releases/build-tools :
```
If you're using Android plugin for Gradle 3.0.0 or higher, your project automatically uses a default version of the build tools that the plugin specifies.
```

Reply:
I meant `targetSdkVersion`, we already set 28 in the manifest.

Reply:
According to this, newer buildTools are backwards compatible with older targetSdk's:
https://stackoverflow.com/a/24523113

Also `AndroidManifest.xml`' values should be removed since gradle ignores them anyway.

Reply:
Should be fixed